website_docs/quick_start.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Ruby Quick Start Guide"
+weight: 24
+description: >
+  <img width="35" src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Ruby_SDK.svg"></img>
+  A language-specific implementation of OpenTelemetry in Ruby.
+---
+
+## Quick Start
+
+[OpenTelemetry for Ruby][repository] can be used to add automatic and manual instrumentation to your applications.
+Automatic instrumentation is enabled by adding [instrumentation packages][auto-instrumentation].
+Manual instrumentation can be added using the [OpenTelemetry API][manual-instrumentation].
+
+### Requirements
+
+These instructions will explain how to set up automatic and manual instrumentation for a Ruby service.
+In order to follow along, you will need:
+
+- Ruby 2.5 or higher
+- Docker Compose
+
+### Installation
+
+The first step add these gems to your Gemfile:
+
+```ruby
+gem 'opentelemetry-sdk'
+gem 'opentelemetry-exporter-otlp'
+gem 'opentelemetry-instrumentation-all'
+```
+
+The inclusion of `opentelemetry-instrumentation-all` in the above list provides instrumentations for several frameworks such as Rails and Sinatra as well as database drivers and HTTP [libraries][auto-instrumentation].
+
+### Initialization
+
+The OpenTelemetry initialization needs to happen early in your application lifecycle.
+For Rails applications, the usual way to initialize OpenTelemetry is in a Rails initializer.
+For other Ruby services, perform this initialization as early as possible in the start-up process.
+
+OpenTelemetry initialization:
+
+```ruby
+# config/initializers/opentelemetry.rb
+require 'opentelemetry/sdk'
+require 'opentelemetry/exporter/otlp'
+require 'opentelemetry/instrumentation/all'
+OpenTelemetry::SDK.configure do |c|
+  c.service_name = '<YOUR_SERVICE_NAME>'
+  c.use_all() # enables all instrumentation!
+end
+```
+
+Now that you have setup your application to perform tracing you'll need to configure the SDK to export the traces to somewhere that will vizualize them for you.
```

Comment:
So ... technically one must configure the SDK to export the traces as part of initialization, not after (unless I've been doing it wrong? a distinct possibility, honestly).

Maybe we could say this:
```suggestion
Now that you have setup your application to perform tracing, you'll need to configure the SDK to export the traces somewhere. Our example loaded the `OTLP` exporter, which the SDK tries to use by default. Next, we'll use the OpenTelemetry Collector to receive these traces and the Jaeger UI to visualize them!
```

That might be too dense, though?

Reply:
How about this?

```suggestion
Now that you have setup your application to perform tracing, you'll need to configure the SDK to export the traces somewhere. Our example loaded the `OTLP` exporter, which the SDK tries to use by default. Next, we'll use the OpenTelemetry Collector to receive these traces and visualize them using Jaeger and Zipkin!
```

Reply:
👍 👍 👍 👍